src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,26 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.1-m1
+
+* `master` branches across all the projects are renamed to `devel`
+* Gateway engine is renamed to `submariner-gateway` from `submariner`
+* Helm charts support submariner deployments using the operator
+* Broker creation is managed by the operator inside of subctl
+* Operator and individual applications have separate service accounts with appropriate privileges
```

Comment:
```suggestion
* Operator and individual Pods have separate service accounts with appropriate privileges
```

Instead of applications, shall we use Pods (or Submariner Pods)

Reply:
```suggestion
* Submariner Pods now have separate service accounts with appropriate privileges
```